### PR TITLE
Feature/create evidence view

### PR DESCRIPTION
**Pull request by JohnPadgettCivica**
If you could give these changes a quick review as would like to get things checked in as quick as possible so other people can help on the screen if necessary.

Early version of the Create evidence view and view model creation along with initial unit tests.

TODO: Validation, Request Handler for saving

### CODE REVIEW

File: src/EA.Prsd.Core.Web.Mvc.Tests/GdsTextboxTests.cs [hunk 1]
```diff
@@ -18,5 +18,21 @@ public void GivenGDSTextBox_ShouldContainGdsCssClass()
 
             control.ToString().Should().Contain("class").And.Contain("govuk-input");
         }
+
+        [Fact]
+        public void GivenGDSTextBox_ShouldContainWidthClass()
+        {
+            var control = htmlHelper.Gds().TextBoxFor(m => m.Nested.Bottom);
+
+            control.ToString().Should().Contain("class").And.Contain("govuk-!-width-one-half");
```
Comment (meglenapnovoselska) [inline src/EA.Prsd.Core.Web.Mvc.Tests/GdsTextboxTests.cs:27]: could be put in a variable and re used.

File: src/EA.Weee.Web.Tests.Unit/RazorHelpers/GdsDropDownUnitTests.cs [hunk 2]
```diff
@@ -64,5 +66,37 @@ public void GivenGDSDropDown_WithoutAutoComplete_ShouldNotContainAutoCompleteCss
 
             control.ToString().Should().NotContain("gds-auto-complete");
         }
+
+        [Fact]
+        public void GivenGDSDropDown_WithoutHalfWidth_ShouldNotContainHalfWidthCss()
+        {
+            var control = htmlHelper.Gds().DropDownListFor(m => m.TopLevel, new List<SelectListItem>(), useHalfWidth: false);
+
+            control.ToString().Should().NotContain("govuk-!-width-one-half");
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Web.Tests.Unit/RazorHelpers/GdsDropDownUnitTests.cs:75]: Moved into a class const...

File: src/EA.Prsd.Core.Web.Mvc.Tests/GdsTextboxTests.cs (hunk 1 above)
Comment (JohnPadgettCivica) [reply src/EA.Prsd.Core.Web.Mvc.Tests/GdsTextboxTests.cs]: Moved into a class const...

File: src/EA.Weee.Core/AatfEvidence/Protocol.cs [hunk 3]
```diff
@@ -0,0 +1,20 @@
+﻿namespace EA.Weee.Core.AatfEvidence
+{
+    using Prsd.Core.Domain;
+
+    public class Protocol : Enumeration
+    {
+        public static readonly Protocol Actual = new Protocol(1, "Actual");
+        public static readonly Protocol NationalProtocol = new Protocol(2, "National protocol");
+        public static readonly Protocol SiteProtocol = new Protocol(2, "Site protocol");
```
[the remark this answers is not in the document]
Comment (JohnPadgettCivica) [reply src/EA.Weee.Core/AatfEvidence/Protocol.cs]: Nope good spot, have updated.

**Pull request merged**